urdf/include/urdf/model.h

```diff
@@ -44,6 +44,7 @@
 #include <tinyxml.h>
 #include <boost/shared_ptr.hpp>
 #include <boost/weak_ptr.hpp>
+#include <ros/ros.h>
```

Comment:
AFAIK in package format 1 "exported build depends" have to be declared as `run_depend`.